app/screens/Settings/SettingsScreen.tsx

```diff
@@ -5,7 +5,7 @@ import {
   ScrollView,
   Image,
 } from "react-native";
-import Constants from "expo-constants";
```

Comment:
fixed existing eslint warning for [import/no-named-as-default](https://github.com/benmosher/eslint-plugin-import/blob/v2.22.1/docs/rules/no-named-as-default.md)
`Using exported name 'Constants' as identifier for default export`